pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotTenantRestletResource.java

```diff
@@ -690,9 +686,46 @@ public SuccessResponse deleteTenant(
   @ApiOperation(value = "Rebalances all the tables that are part of the tenant")
   public TenantRebalanceResult rebalance(
       @ApiParam(value = "Name of the tenant whose table are to be rebalanced", required = true)
-      @PathParam("tenantName") String tenantName, @ApiParam(required = true) TenantRebalanceConfig config) {
+      @PathParam("tenantName") String tenantName,
+      @ApiParam(value = "Number of table rebalance jobs allowed to run at the same time", required = true, example =
+          "1")
+      @QueryParam("degreeOfParallelism") Integer degreeOfParallelism,
+      @ApiParam(value =
+          "Comma separated list of tables (with OFFLINE or REALTIME suffix) that are allowed in this tenant rebalance"
+              + " job. Leave blank to allow all tables from the tenant", example = "")
+      @QueryParam("allowTables") String allowTables,
```

Comment:
can we call this `includeTables` instead? feels like it makes the intent clearer